lib/obs.rb

```diff
@@ -151,7 +152,34 @@ def self.search_published_binary(query, opts = {})
       return [] if result.binaries.nil?
 
       result.binaries.map! { |bin| OBS.add_project_quality(bin) }
-      result.binaries.map { |bin| OBS.add_fileinfo_to_binary(bin) }
+      result.binaries.map! { |bin| OBS.add_fileinfo_to_binary(bin) }
+      result.binaries.map! { |bin| OBS.add_binary_relevance(bin, query) }
+
+      result.binaries.sort { |a, b| b.relevance <=> a.relevance }
```

Comment:
use `sort_by(:relevance).reverse` or `sort_by { |bin| - bin.relevance }`, it is more efficient :wink:

Reply:
I tried to use `sort_by(&:relevance).reverse`, but it did not work - the order is not the same and as a result `rails test:system` failed. `sort_by { |bin| - bin.relevance }` works fine, so I am using that. Thanks!

Reply:
That's probably because some elements have the same relevance.

Take into account that both `sort` and `sort_by` are not guarantee the results to be stable. That means that the fact that they are ordered as you want could change in a future Ruby release. I think we should handle the relevance in a better way. But this can be done in another PR.